site/src/pages/release-notes/1.20.0.mdx

```diff
@@ -0,0 +1,150 @@
+---
+date: 2022-10-05
+---
+
+## 🌟 New features
+
+- You can now convert the GraphQL Error into an <type://HttpResponse> using <type://GraphqlErrorHandler>. #4435
+  ```java
+  GraphqlErrorHandler errorHandler = new GraphqlErrorHandler() {
+    @Override
+    public HttpResponse handle(ServiceRequestContext ctx, ExecutionInput input,
+                               ExecutionResult result, @Nullable Throwable cause) {
+      final List<GraphQLError> errors = result.getErrors();
+      if (errors.stream().map(GraphQLError::getMessage)
+                .anyMatch(m -> m.startsWith("Unauthorized"))) {
+        return HttpResponse.of(HttpStatus.UNAUTHORIZED);
+      }
+      ...
+    }
+  };
+  GraphqlService.builder()
+                .schemaFile(...)
+                .runtimeWiring(...)
+                .errorHandler(errorHandler)
+                .build();
+  ```
+
+- You can now easily configure client-side TLS key and certificates to support mTLS via
+  <type://ClientFactoryBuilder#tls(File,File)>. #4410
+  ```java
+  File keyCertChainFile = ...;
+  File keyFile = ...;
+  ClientFactory factory =
+    ClientFactory.builder()
+                 .tls(keyCertChainFile, keyFile)
+                 .build();
+  WebClient.builder()
+           .factory(factory)
+           .build();
```

Comment:
Global comment: Should we break the line before `.builder` since we decide to use more aggressive indentation with 2 spaces for release notes. https://github.com/line/armeria/blob/26e57a871d496e7fc9e135bbf86225b74c708ab8/site/src/pages/release-notes/1.18.0.mdx#L43-L48
```suggestion
  ClientFactory factory =
    ClientFactory
      .builder()
      .tls(keyCertChainFile, keyFile)
      .build();
  WebClient
    .builder()
    .factory(factory)
    .build();
```